lstchain/scripts/lstchain_merge_hdf5_files.py

```diff
@@ -22,6 +22,22 @@
 
 from lstchain.io import auto_merge_h5files
 from lstchain.io import get_dataset_keys
+from lstchain.io.io import (
+    dl1_params_tel_mon_ped_key,
+    dl1_params_tel_mon_cal_key,
+    dl1_params_tel_mon_flat_key,
+    dl1_mon_tel_CatB_cal_key,
+    dl1_mon_tel_CatB_ped_key,
+    dl1_mon_tel_CatB_flat_key
+)
+
+default_keys_to_copy = [dl1_params_tel_mon_ped_key, 
```

Comment:
The test is currently failing because the merging function expects `CatB` keys that are not present in the test h5 files. The function should check first whether they exit